src/services/ShSimulatorService.tsx

```diff
@@ -146,6 +146,25 @@ const ShSimulation = ({ children }: GenericContextProviderProps) => {
 		[authKy]
 	);
 
+	/**
+	 * Returns the endpoint based on the simulation info.
+	 * If the platform is not known, it will return 'jobs/direct'.
+	 * The platform name is stored in metadata of the simulation info and is typed by `PlatformType`.
+	 * Endpoints are named after the platform types.
+	 * @see PlatformType
+	 * @param info - The simulation info object.
+	 * @returns The endpoint string.
+	 */
+	const getEndpointFromSimulationInfo = (info: SimulationInfo) => {
+		const platform =
+			`jobs/${info.metadata.platform.toLowerCase()}` as `jobs/${Lowercase<PlatformType>}`;
+
+		if (platform in ['jobs/direct', 'jobs/batch']) return platform;
+		console.error(`Simulation platform is unknown: ${info.metadata.platform}`);
```

Comment:
It happens that this code fails (i.e. batch simulation + manual refresh of job status), the code fails with:

Simulation platform is unknown: BATCH

That suggests tha the logic above is somehow wrong,

Reply:
`in` is a keyword for objects mistakenly used here to search array
I will replace it with includes
![image](https://github.com/yaptide/ui/assets/26445545/46799890-f486-4e0f-bf69-cfe53e7af730)